packages/card-picker/package.json

```diff
@@ -42,12 +42,13 @@
     "ember-cli-sri": "^2.1.1",
     "ember-cli-template-lint": "^1.0.0-beta.1",
     "ember-cli-uglify": "^2.1.0",
+    "ember-data": "^3.5.0",
```

Comment:
why is Ember Data needed for this PR?

Reply:
When you open the bottom edge it (eventually) instanciates the `cardstack-query-editor` from `@cardstack/search` which uses the `ember-data` store.

Looking at the `@cardstack/search` it seems that `ember-data` is not a dependency 🤔 I guess that should be changed.

@ef4 or @habdelra any idea why we're using a particular sha as an ember-data dependency on `@cardstack/search`? https://github.com/cardstack/cardstack/blob/master/packages/search/package.json#L44

Reply:
I’m unsure, @ef4 would have to answer that.